test/new-e2e/pkg/environments/aws/ecs/ecs.go

```diff
@@ -164,6 +166,17 @@ func WithoutAgent() ProvisionerOption {
 	}
 }
 
+// WorkloadAppFunc is a function that deploys a workload app to a kube provider
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
// WorkloadAppFunc is a function that deploys a workload app to an ECS cluster
```